src/kixi/mailer/ses.clj

```diff
@@ -2,39 +2,43 @@
   (:require [amazonica.aws.simpleemail :as email]
             [com.stuartsierra.component :as component]
             [clostache.parser :as parser]
-            [clojure.spec :as s]
+            [clojure.spec.alpha :as s]
             [kixi.comms :as c]
             [kixi.log.timbre.appenders.logstash :as l]
             [taoensso.timbre :as timbre :refer [error]]
             [clojure.java.io :as io]))
 
 (defn invalid-rejected
   [cmd explaination]
+(prn cmd)
```

Comment:
Do you really want a print statement here? Are we not logging things with timbre etc?

Reply:
debug left behind ;)